node/src/validator/mod.rs

```diff
@@ -142,7 +142,7 @@ impl<N: Network, C: ConsensusStorage<N>> Validator<N, C> {
             node.rest = Some(Rest::start(rest_ip, Some(consensus), ledger, Arc::new(node.clone()))?);
         }
         // Initialize the sync pool.
-        node.initialize_sync()?;
+        // node.initialize_sync()?;
```

Comment:
For now this is okay, however we should keep this operational in case a node's quic goes down, or in case there are byzantine actors - the sync protocol will help to disconnect the byzantine validators.